docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: https://github.com/Luzifer/ots.git#v1.8.0
+    restart: always
+    env_file:
+      - .env
```

Comment:
Lets please call this `docker-compose.env` or something as `.env` for example clashes with my dev-env - and I'm sure with many other devs too as `.env` is a common dev-file which should **never** be commited as it contains secrets. 
(In my case that file is ignored in global git-ignore and also raise a security alarm and I will no longer be able to push to this repo as long as it is committed… - Too many too scary things in `.env` files…)

Reply:
There are only few entries in `.env` file, wouldn't it be better to just explicitly include them in `environment` section?